src/components/common/panels/panels.tsx

```diff
@@ -0,0 +1,39 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import * as styles from './panels.module.scss';
+
+type Panel = {
+  head: string;
+  body: string;
+};
+type Props = {
+  panels: Panel[];
+  columns?: number;
+  bodySize?: 'small' | 'regular';
+};
+
+const Panels: React.FC<Props> = ({
+  panels,
+  columns = 2,
+  bodySize = 'regular',
+}) => {
+  return (
+    <div className={styles.wrapper} data-columns={columns}>
```

Comment:
```suggestion
    <div className={styles.wrapper}>
```

Reply:
Any reasons? It's convenient since there are different columns grid on the page.

Reply:
Don't like this approach, will need to think how to get rid of it, let's leave for now, but we will definitely remove this in the future